crypto/keys_test.go

```diff
@@ -42,3 +42,32 @@ func TestPublicKeyHex(t *testing.T) {
 	}
 	assert.Equal(t, 128, len(key.Hex()))
 }
+
+func benchmarkDifficultKeyGeneration(difficulty uint, b *testing.B) {
+    for n := 0; n < b.N; n++ {
+         GenerateDifficultKey(difficulty)
+    }
+}
+
+// $ go test  -bench=. -benchtime=1m ./crypto/
+// goos: darwin
+// goarch: amd64
+// BenchmarkDifficultKeyGeneration14-8   	      50	2720878068 ns/op
+// BenchmarkDifficultKeyGeneration16-8   	      50	7893233868 ns/op
```

Comment:
I've followed https://dave.cheney.net/2013/06/30/how-to-write-benchmarks-in-go

Reply:
I'm used to put sample results to be able to easily recover it w/o need to run time-consuming benchmarks

Reply:
Banchmark results without any reference to on which hardware they were run aren't really that profitable. For me it just adds noise to the code.
Referring to golang itself again: https://github.com/golang/go/blob/master/src/sort/sort_test.go
no benchmark results in the comments.

Reply:
well, here I'll insist to leave it.

Reply:
Since this is a fresh repo I will try to enforce and manage some code style guidelines and conventions to make it look like a single team is working on the project, not a bunch of people who happen to commit to the same repository. And I feel that if everyone start putting their own unstandarized hardware specs and results for benchmarks in the comments it's closer to the latter rather to the former.
Instead I recommend writing a godoc style comment (https://blog.golang.org/godoc-documenting-go-code) for the `GenerateDifficultKey` function describing the expectations using natural language. Like saying that on modern CPU generation of a key with difficulty of 14 should take around 1 second.

Reply:
I've added `godoc`, but still insist to leave this benchmark results in comments. Please.

It is a fresh repo in deed, so we can introduce our own standards for benchmarks comments, and I propose this. I always threat benchmark code as some sort of freestyle. No tests nor production code depends on benchmarks. 

This is my final statement. If you cannnnot agree, than I'll remove it with sorrow.